src/js/app.js

```diff
@@ -91,6 +91,49 @@ $(document).ready(function() {
     }
   });
 
+  //Add toggle effect for sliding #menu
+  let showingMenu = false;
+  const $menu = $('#menu');
+  const $main = $('main');
+  const config2 = {
+    duration: 1000,
+    easing: 'easeOut'
+  };
+  $('#menu-burger,.close-menu,#navigation-links>li>a').on('click', function() {
+    if (showingMenu) {
+      $menu.removeClass('active');
+      $main.removeClass('menu-active');
+      showingMenu = false;
+      event.preventDefault();
+    } else {
+      $menu.addClass('active');
+      $main.addClass('menu-active');
+      showingMenu = true;
+      event.preventDefault();
+    }
+    return showingMenu;
```

Comment:
Haha funny man. Returning ends the function. So whatever code you write behind it does not get executed. Return is used to give back a variable:
```
function fullName(firstName, lastName) {
  return 'Sir ' + firstName + ' ' + lastName';
}
const jan = fullName('Jan', 'Werkhoven');
// Sir Jan Werkhoven
```
So the idea of functions is that you 1) put variables in, 2) process them and 3) return them. You can return arrays, objects, strings, booleans, ...